OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitYoutubePlayerFragment.java

```diff
@@ -114,8 +114,18 @@ protected long getPlayerCurrentPosition() {
 
     @Override
     protected void setFullScreen(boolean fullscreen) {
-        if (youTubePlayer != null) {
-            youTubePlayer.setFullscreen(fullscreen);
+        /* Some times `YouTubePlayerSupportFragment` throws the `IllegalStateException` when user moves
```

Comment:
```
        /*
         * If the youtube player is not in a proper state than it throws the  IllegalStateException.
         * To avoid the crash and continue the flow we are reinitializing the player here.
         * 
         * It may occur when the edX app was in background and user kills the device youtube app
         */

```
It seems better comment.

Reply:
updated.